src/test/java/sync/google/GoogleKeywordDownloaderTest.java

```diff
@@ -34,6 +39,64 @@ public void setup() {
     }
 
     @Test
+    public void testDownloadTwoGoogleKeywordsBelongsToSameCampaignId(){
+        setupKeywordsInGoogle(
+                keyword().withId(1).withCampaignId(CAMPAIGN_ID).withStatus("Active").withText("shoes"),
+                keyword().withId(1).withCampaignId(CAMPAIGN_ID).withStatus("Paused").withText("shirts")
+        );
+
+        List<KenshooKeyword> kenshooKeywords = testedDownloader.download(
+                    selector().withCampaignIdInTarget(CAMPAIGN_ID).build()
+        );
+
+        assertThat(kenshooKeywords, hasSize(2));
+    }
+
+    @Test
+    public void testDownloadActiveGoogleKeyword(){
```

Comment:
The test name, if you split it into words, should be a sentence explaining the business requirement.
In this case:
> test that Active status in Google is mapped to Active status in Kenshoo
It sounds a trivial requirement for the 'Active' case, but its good.
Another example could be:
> test that BlaBlaBla status in Google is mapped to Unknown status in Kenshoo